addOns/custompayloads/src/main/java/org/zaproxy/zap/extension/custompayloads/CustomPayloadsParam.java

```diff
@@ -38,17 +38,14 @@ public class CustomPayloadsParam extends AbstractParam {
             CUSTOM_PAYLOADS_BASE_KEY + ".categories.category";
     private static final String CATEGORY_NAME_KEY = "[@name]";
 
-    private static final String PAYLOAD_ID_KEY = "id";
     private static final String PAYLOAD_KEY = "payload";
     private static final String PAYLOAD_ENABLED_KEY = "enabled";
 
     private static final String CONFIRM_REMOVE_PAYLOAD_KEY =
             CUSTOM_PAYLOADS_BASE_KEY + ".confirmRemoveToken";
-    private static final String NEXT_PAYLOAD_ID_KEY = CUSTOM_PAYLOADS_BASE_KEY + ".nextPayloadId";
```

Comment:
This should be removed from the config and all IDs. too

Reply:
You mean, roll the param version and add a method that trims this stuff?

Reply:
Yes.

Reply:
Actually.
1) It isn't a versioned config.
2) Won't they be removed when saved?

Reply:
1. Yes, you need to change it to be.
2. This one definitely wouldn't. The others eventually but that's no reason to not clear whatever is no longer used when updating the add-on.